### ISSUE TITLE
Kent noted that links to old releases are broken.

### ISSUE THREAD

**Issue by jgallagher59701** [OPENDAP/Website#51]

**Comment by Mikejmnez** [comment]
Indeed when I click on the option `Hyrax-1.16` (see screenshot below) at the bottom of the (most) [recent release page](https://www.opendap.org/hyrax-1-17/), it takes me to the page [Hyrax Data-Server](https://www.opendap.org/software/hyrax-data-server/).

**Comment by jgallagher59701** [comment]
Right, here's what he sent me in slack: Go to https://www.opendap.org/hyrax-1-17/ and scroll down close to the end of the page. You see "Older Versions" with Hyrax-1.16. Click Hyrax-1.16, you don't see the older releases but it comes to the top hyrax release page.

**Comment by Mikejmnez** [comment]
I found [this](https://old.opendap.org/pub/binary/) location in the old website where binaries for older releases can be found (among other things like handlers). You think it would be fine to add the specific links for those releases on the website? Something like 

## Previous Releases

[Hyrax-1.16](https://old.opendap.org/pub/binary/hyrax-1.16/)
[Hyrax-1.15](https://old.opendap.org/pub/binary/hyrax-1.15/)

and so on

**Comment by jgallagher59701** [comment]
That's the stuff.

Lets use those URLs. If you go up a level to https://old.opendap.org/pub/ you will see all the old binaries and sources we have.  You could make a set links for the 1.16 release binaries and then add two separate links for all the old binaries and sources. For now, we pay for that site and it hosts a lot of stuff.

I tested the site and it's pretty fast. For ~$60 year, it seems like a deal. We can change the name from 'old.opendap.org' something more snazzy like 'software.opendap.org' or whatever ChatGPT suggests ;-)

**Comment by Mikejmnez** [comment]
Sounds like a plan.

**Comment by Mikejmnez** [comment]
Ok, this is work in progress. I have updated the Hyrax release post to show links to the older versions to binaries and source. But I am not super familiar with the exact files needed to download/install Hyrax 1.16 correctly. There are various options for Hyrax 1.16 and I don't know which one of this is the correct one (see https://old.opendap.org/pub/binary/hyrax-1.16/)

Below is a screenshot from the bottom of the [official hyrax 1.17 release ](https://www.opendap.org/hyrax-1-17/)

![Screenshot 2024-09-16 at 11 04 09 AM](https://github.com/user-attachments/assets/00b12753-4d2f-406e-b3f3-9ef909d3b9cc)

**Comment by Mikejmnez** [comment]
After talking with Nathan, I will recreate the Download Hyrax from previous versions taking the template from [Download Hyrax-1.17](https://www.opendap.org/hyrax-1-17/) for each of the previous releases, instead of making available all the pub directory.

**Comment by Mikejmnez** [comment]
I created a new post about [Download Hyrax 1.16](https://www.opendap.org/hyrax-1-16/) with date matching that of the release of 1.16.18. 

I also included a link to the at the bottom of [Download Hyrax-1.17.0](https://www.opendap.org/hyrax-1-17/) that takes you there. 

![Screenshot 2024-09-18 at 5 31 23 PM](https://github.com/user-attachments/assets/9604d51d-079d-45bf-bf9c-b9e89413da3a)

Regarding the content of  [Download Hyrax 1.16](https://www.opendap.org/hyrax-1-16/)

- I did not include the "Installing the Binary Distribution" associated with Linux (RHEL-8) binaries. I could not a copy-past from Hyrax-1.17.0, but I wasn't sure if some of the specific information would be different in 1.16.8.
